sdks/java/io/csv/src/main/java/org/apache/beam/sdk/io/csv/CsvIO.java

```diff
@@ -55,6 +55,68 @@
  * <p>Reading from CSV files is not yet implemented. Please see <a
  * href="https://github.com/apache/beam/issues/24552">https://github.com/apache/beam/issues/24552</a>.
  *
+ * <h3>Valid CSVFormat Configuration</h3>
+ *
+ * <p></>A <a
```

Comment:
this list sounds very specific, imo it's better to put it into the javadoc of CsvIO.Read than top level javadoc here. For now we can put it into the javadoc of CsvIOParseHelper.validate, as this defines the spec of validate method.

@damondouglas what do you think?

Reply:
@Abacn Balid CSVFormat configuration details apply to both ReadFiles and Parse so I like where it is.